breathecode/tests/mixins/generate_models_mixin/marketing_models_mixin.py

```diff
@@ -54,6 +56,20 @@ def generate_marketing_models(self,
                 **automation_kwargs
             })
 
+        if not 'downloadable' in models and is_valid(downloadable):
+            kargs = {}
+
+            if 'academy' in models:
```

Comment:
`Academy` have a mandatory relationship, make sure if a `Downloadable` is being generated, also create the academy if `is_valid(downloadable) == True`